src/services/supabaseClient.ts

```diff
@@ -0,0 +1,6 @@
+import { createClient } from "@supabase/supabase-js";
+
+const supabaseUrl = process.env.SUPABASE_URL;
+const supabaseKey = process.env.SUPABASE_ANON_KEY;
+
+export const supabase = createClient(supabaseUrl!, supabaseKey!);
```

Comment:
```suggestion
if (!supabaseUrl || !supabaseKey) {
  throw new Error("Missing Supabase config vars");
}

export const supabase = createClient(supabaseUrl, supabaseKey);
```

Safer and less vague in TS.